AOEMods.Essence.Editor/MainViewModel.cs

```diff
@@ -196,11 +196,18 @@ private void OpenStreamWithViewerFromExtension(Stream stream, string extension,
                 Trace.WriteLine($"Remaining data: {stream.Length - stream.Position:X} at {stream.Position:X}");
                 break;
             default:
-                MessageBox.Show(
-                    $"Unsupported extension '{extension}'", "Unsupported extension",
-                    MessageBoxButton.OK, MessageBoxImage.Error
-                );
+                var file = Path.GetTempPath() + title;
+                using (var fileStream = File.OpenWrite(file))
+                {
+                    stream.CopyTo(fileStream);
+                }
+                Process proc = new Process();
+                proc.EnableRaisingEvents = false;
+                proc.StartInfo.FileName = Path.Combine(Environment.SystemDirectory, "rundll32.exe");
+                proc.StartInfo.Arguments = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), "shell32.dll") + ",OpenAs_RunDLL " + file;
+                proc.Start();

```

Comment:
This is for opening the file with the default application I assume? I'd prefer this way, we could move it into a utility

https://github.com/aoemods/AOEMods.Essence/blob/2c63983f03d9f82b9ecffc89b9abcb17eb324cb0/AOEMods.Essence.Editor/MainViewModel.cs#L209-L222